engine/src/test/java/com/redhat/cloud/notifications/templates/EmailTemplateMigrationServiceTest.java

```diff
@@ -112,18 +112,6 @@ void testMigration() {
         EventType operatorDataProcessed = resourceHelpers.createEventType(costManagement.getId(), "cm-operator-data-processed");
         EventType operatorDataReceived = resourceHelpers.createEventType(costManagement.getId(), "cm-operator-data-received");
 
-        /*
-         * Bundle: application-services
-         */
-        Bundle applicationServices = resourceHelpers.createBundle("application-services");
-        // App: rhosak
-        Application rhosak = resourceHelpers.createApp(applicationServices.getId(), "rhosak");
-        EventType scheduledUpgrade = resourceHelpers.createEventType(rhosak.getId(), "scheduled-upgrade");
-        EventType disruption = resourceHelpers.createEventType(rhosak.getId(), "disruption");
-        EventType instanceCreated = resourceHelpers.createEventType(rhosak.getId(), "instance-created");
-        EventType instanceDeleted = resourceHelpers.createEventType(rhosak.getId(), "instance-deleted");
-        EventType actionRequired = resourceHelpers.createEventType(rhosak.getId(), "action-required");
-
         /*
          * Bundle: ansible
```

Comment:
Bundle `ansible` ? On the other comment I saw that `ansible` was an application - maybe we need to check if there is something wrong?

Reply:
Good questions.

This looks like unfinished business and we should contact whoever added the `ansible` instant template to check if that still makes sense.

Reply:
https://github.com/RedHatInsights/notifications-backend/pull/823

Reply:
```
* Suggested changes for ansible:
 - Remove code from aggregator
 - Use ansible as the bundle and reports as the app +
 - Add instant body and title
```